imagepuller/internal/service/verify_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2025 Edgeless Systems GmbH
+// SPDX-License-Identifier: BUSL-1.1
+
+package service
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestImagePullerService_getAndVerifyImage(t *testing.T) {
+	log := slog.New(slog.NewTextHandler(os.Stderr, &slog.HandlerOptions{
+		Level: slog.LevelInfo,
```

Comment:
iirc there is a test logger

Reply:
We don't have a test logger right now, but I guess we should have one eventually? Other unit tests tend to use `slog.Default()`, maybe consider doing that here, too, so we'll find it easier when we add a test logger.